### PR TITLE
Sunbird RC: Starter pack for location master

### PR DESCRIPTION
**Pull request by anmode**
## Project

Starter pack for location master

### Changes

What was updated (new pages, weekly update, etc.)

### Notes

If there are any further notes.

### CODE REVIEW

File: docs/2023/Sunbird RC/Starter pack location master/index.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+---
+
+
+# Author
+Anmol Agrawal
+
+# Contact Info
+[Email](mailto:anmol2002.aa1@gmail.com)
+[Linked In](www.linkedin/in/anmode)
+[GitHub](www.github.com/anmode)
```
Comment (photon0205) [inline docs/2023/Sunbird RC/Starter pack location master/index.md:14]: add https links instead of this in linkedin and github

**Review by photon0205 — CHANGES_REQUESTED** [review]
change the author in every week's file to your name

**Review by photon0205 — APPROVED** [review]

**Comment by anmode** [comment]
thanks @photon0205

File: docs/2023/Sunbird RC/Starter pack location master/index.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+---
+
+
+# Author
+Anmol Agrawal
+
+# Contact Info
+[Email](mailto:anmol2002.aa1@gmail.com)
+[Linked In](https://www.linkedin.com/in/anmode/)
+[GitHub](https://www.github.com/anmode)
```
Comment (claddyk) [inline docs/2023/Sunbird RC/Starter pack location master/index.md:14]: nit:
```bash
- [Email](mailto:anmol2002.aa1@gmail.com)
- [Linked In](https://www.linkedin.com/in/anmode/)
- [GitHub](https://www.github.com/anmode)
```

**Review by claddyk — COMMENTED** [review]
Lgtm

**Pull request merged**